spec/abstractops.html

```diff
@@ -157,10 +219,25 @@ <h1>CreateDateTime ( …args )</h1>
     </p>
   </emu-clause>
 
-  <emu-clause id=sec-temporal-ToTimeLike aoid=ToTimeLike>
-    <h1>ToTimeLike ( _time_ )</h1>
-    <p>
-      <mark>TODO: Define.</mark>
-    </p>
+  <emu-clause id=sec-temporal-FormatSecondsStringPart aoid=FormatSecondsStringPart>
+    <h1>FormatSecondsStringPart ( _second_, _millisecond_, _microsecond_, _nanosecond_ )</h1>
+    <emu-alg>
+      1. If _second_, _millisecond_, _microsecond_, and _nanosecond_ are all 0, then
+        1. Return `""`.
+      1. Let _nanos_, _micros_, and _millis_ be `""`.
```

Comment:
Do we typically set more than one variable at a time like this?

Reply:
I'm not sure. I can make it more verbose if you prefer.